tests/InlineTextTest.php

```
@@ -7,9 +7,49 @@ class InlineTextTest extends CopilotTagTest
     public function expectedWrites()
     {
         return [
+            [
+                new InlineText(),
```

This should throw an Exception. Maybe you need to rebase this branch?